cerebrium/fine-tuning/diffusion-models/config.mdx

```diff
@@ -0,0 +1,142 @@
+
+In this section, we introduce the parameters you may want to manipulate for your training, however, we'll set the defaults to what we've found works well if you'd like to leave them out!
+
+If you would like to override the parameters in your config file during a deployment, the `--config-string` option in the command line accepts a JSON string. The parameters provided in the JSON string will override the values they may have been assigned in your config file.
+
+For your convenience, an example of the config file is available [here](#example-yaml-config-file)
+
+## Setting up a config file
+
+Your yaml config file can be placed anywhere on your system, just point the trainer to the file.
+
+In your YAML file, you need to include the following required parameters or parse them into the `cerebrium train` command:
+
+### Required parameters
+| Parameter               | Suggested Value                | Description                                                                          |
+| ----------------------- | ------------------------------ | ------------------------------------------------------------------------------------ |
+| training_type           | diffuser                       | Type of training to run. Either diffuser or transformer. In this case, use diffuser. |
+| name                    | test-config-file               | Name of the experiment.                                                              |
+| api-key                 | Your private API key           | Your Cerebrium API key.                                                              |
+| hf_model_path           | runwayml/stable-diffusion-v1-5 | ---                                                                                  |
+| train_prompt            | Photo of a tsdf dog.           | ---                                                                                  |
+| log_level               | INFO                           | log_level level for logging. Can be DEBUG, INFO, WARNING, ERROR.                     |
+| train_image_dir         | data/training_class_images/    | Directory of training images.                                                        |
+
+#### Optional parameters
+
+Additionally, you can include the following parameters in your config file:
+
+| Parameter                        | Suggested Value         | Description                                                                |
+| -------------------------------- | ----------------------- | -------------------------------------------------------------------------- |
+| **Diffuser Parameters** |                         |                                                                            |
```

Comment:
spacing of the |'s if off on this line

Reply:
does it matter?

Reply:
Doesn't matter but I'll fix for readability